Makefile

```diff
@@ -142,6 +142,10 @@ test: lint karma
 runserver: remove_old_config
 	$(REPO_BIN_DIR)/run-server.sh
 
+.PHONY: runfx
+runfx:
+	lib/run.js ${ARGS}
```

Comment:
We should probably do something similar to distserver where it calls node explicitly for server.js, so no need for the #! node. Similarly let's keep run/util.js in bin/ instead of lib/.